### ISSUE TITLE
OS Monterey support, intel mac, virtual box

### ISSUE THREAD

**Issue by dstorozhuk** [docksal/docksal#1595]
**Description**

VB on OS Monterey works only after 6.1.29 version, as I am aware. 
But, the installation instructions recommends to use https://docksal.io/installation#macos-virtualbox 6.1.22 version. 
So the VB 6.1.29/6.1.30 on OS Monterey does not supported yet?

**Comment by 610** [comment]
@dstorozhuk I had the same issue.
This worked for me.
Update to 6.1.30 latest VBox version, then create the following file in `/etc/vbox/networks.conf` with the following content:
```
* 0.0.0.0/0 ::/0
```

Reboot your Mac and it should work.

**Comment by dstorozhuk** [comment]
@610 no issues with permission to .kext extensions?

**Comment by 610** [comment]
@dstorozhuk I didn't have any issues with kext. What issue do you have? Is it VirtualBox?

_Sorry, I didn't get notification from github for this comment._

**Comment by lmakarov** [comment]
Closing in favor of https://github.com/docksal/docksal/issues/1596
